src/cltkv1/wordnet/wordnet.py

```diff
@@ -0,0 +1,2288 @@
+
+        form = form.translate(punctuation)
+        if form:
+            results = requests.get(
+                f"{self.host()}/lemmatize/{form}/{morpho if morpho else ''}?format=json",
+                timeout=(30.0, 90.0),
+            )
+            if results and results.json():
+                return (
+                    Lemma(
+                        self,
+                        result["lemma"]["lemma"],
+                        result["lemma"]["morpho"][0],
+                        result["lemma"]["morpho"],
+                        result["lemma"]["uri"],
+                    )
+                    for result in results.json()
+                )
+        return []
+
+    #############################################################
+    # Translater
+    #############################################################
+    def translate(self, language: str, form: str, pos: str = "*"):
+        """
+        Translates an English, French, Spanish, or Italian word into Latin.
+        :param language: 'en', 'fr', 'es', 'it' indicating the source language
+        :param form: The word to translate
+        :param pos: Optionally, a part-of-speech ('n', 'v', 'a', 'r') indicator
+        used as a filter
+        :return: A list of Lemma objects
+
+        >>> LWN = WordNetCorpusReader(iso_code="lat")
+        >>> offspring_translations = list(LWN.translate('en', 'offspring'))
+        >>> print('pusio' in [lemma.lemma() for lemma in offspring_translations])
+        True
+
+        """
+        pos = f"{pos}/" if pos else ""
+        results = requests.get(
+            f"{self.host()}/translate/{language}/{form}/{pos}?format=json",
+            timeout=(30.0, 90.0),
+        )
+        if results:
+            data = results.json()["results"]
+        return (
+            Lemma(self, lemma["lemma"], lemma["pos"], lemma["morpho"], lemma["uri"])
+            for lemma in data
+        )
+
+
+######################################################################
+# WordNet Information Content Corpus Reader
+######################################################################
+class WordNetICCorpusReader(CorpusReader):
+    """
+    A corpus reader for the WordNet information content corpus.
+    :param root: The root directory where the information content file is stored.
+    :param fileids: A list of file names, relative to the root directory, in this
+        case a single file containing information content for a corpus.
+
+    >>> from cltkv1.wordnet.wordnet import WordNetICCorpusReader
+    >>> LWNIC = WordNetICCorpusReader(iso_code='lat', fileids=['ic-lasla.dat'])
+
+    """
+    def __init__(
+        self,
+        iso_code,
+        root=None,
+        fileids=None,
+    ):
+        if not root:
+            iso_map ={'lat': 'latin', 'grk': 'greek', 'skt': 'sanskrit'}
+            root = os.path.join(
+            get_cltk_data_dir(), f"{iso_map[iso_code]}/model/{iso_map[iso_code]}_models_cltk/semantics/wordnet/"
```

Comment:
I see what went wrong. In my doctest for ``, I create a custom OS variable (`$CLTK_DATA`) but do not delete it. Will add this now to master, then I'll update this branch against the new master.

https://github.com/cltk/cltkv1/blob/9f66f72fcc64a6b42a5acebbdaa093b1cd5cca9a/src/cltkv1/utils/utils.py#L115

Reply:
Fixed here https://github.com/cltk/cltkv1/commit/d5ea3693336a1dda330894ec7e5c9e7172d7d5ae